pages/common/ngs.md

```diff
@@ -0,0 +1,16 @@
+# ngs
+
+> Scripting language created specifically for Ops.
+> More information: <https://github.com/ngs-lang/ngs>.
+
+- Execute a code snippet:
+
+`ngs -e "{{echo('ngs is executed')}}"`
+
+- Execute a script:
+
+`ngs "{{path/to/script.ngs}}"`
```

Comment:
Wait, we don´t  use quotes for paths
```suggestion
`ngs {{path/to/script.ngs}}`
```

Reply:
But what if script name contains spaces? It will be treated as distinct arguments instead of one file name. I think it's better to use double quotes where we can. ;)

Reply:
The thing to remember here is that we don't want to document shell builtins. The purpose of tldr pages is to document specific commands. To this end, quoting things that contain spaces I think we can assume is common knowledge, so we don't need to unnecessarily complicate pages by including the syntax here.

Reply:
> - When they are required for a command to function correctly
> - When an argument will always contain spaces.

The problem is `ngs` will fail to run script if it's name is unquoted and contains spaces. According to [your message](https://github.com/tldr-pages/tldr/issues/7521#issuecomment-991406076) I have to quote script path.